imexam/connect.py

```diff
@@ -50,20 +59,45 @@ class Connect(object):
 
     """
 
-    def __init__(self, target=None, path=None, viewer="ds9", wait_time=10, quit_window=True):
+    def __init__(self, target=None, path=None, viewer="ds9",
+                 wait_time=10, quit_window=True):
+
+        # better dynamic way so people can add their own viewers?
+        _possible_viewers = ["ds9"]
```

Comment:
Yeah, this is obv. fine for now, but it would be interesting to extend this in the future.